paket.dependencies

```diff
@@ -1,7 +1,11 @@
-framework: >=net45
+framework: netstandard2.0, netcoreapp2.0, net462
```

Comment:
We only have three target frameworks and so if we list them here lockfiles and such get a bit more sane.
In addition, I bumped us up to net462 since I got a lot of errors around referencing System.Object and System.Runtime on net45. This i think is the most contentious change, and I'd be glad to discuss more.

Reply:
Sorry, I'd like to stay in net45 as the min supported version for a while longer, can you revert that part of the change?